boards/omnibus/f4sd/src/board_config.h

```diff
@@ -228,7 +228,7 @@
 #define BOARD_HAS_PWM    DIRECT_PWM_OUTPUT_CHANNELS
 
 /* This board provides a DMA pool and APIs */
-#define BOARD_DMA_ALLOC_POOL_SIZE    5120
+#define BOARD_DMA_ALLOC_POOL_SIZE (5120 + 512 + 1008)	// 5120 fat + 512 + 1008 spi
```

Comment:
Limit to 1008 (or 1024 perhaps)? We never transfer anywhere near the full FIFO, but the driver still (lazily) allocates the FIFO size. If memory is tight we could be more careful.

```suggestion
#define BOARD_DMA_ALLOC_POOL_SIZE (5120 + 1008)	// 5120 fat + 1008 spi
```